puppet.py

```diff
@@ -27,6 +27,22 @@ def check_disabled():
         sudo('test -f {0} && echo DISABLED || echo ENABLED'.format(lockfile))
 
 @task
+def regenerate_certificate():
+    """Regenerate a node certificate"""
+    if len(env.hosts) > 1:
+      abort("This task should only be run on one host at a time.")
+    sudo('rm -f /etc/puppet/ssl/certs/{}.pem'.format(env.hosts[0]))
+    with settings(host_string='puppetmaster-1.management.production'):
+      sudo('puppet cert clean {}'.format(env.hosts[0]))
```

Comment:
We'll have plans to open up our Puppet repo soon, which will make that viewable by anyone.

`govuk_node_clean` is useful in that in runs all of the actions required to remove a node from Puppet, and also checks that the node being cleaned with already known to Puppet.